### PR TITLE
chore: remove duplicates & exposed data

### PR DESCRIPTION
**Pull request by eedygreen**
## Description
- Removed Duplicates Runners
- Used single Runner to execute previous 5 runners as  5 Jobs - still parallel executions and less time
- Removed exposed data

## Related Issue Or Context
[Issues 25 on workflow improvement](https://app.zenhub.com/workspaces/sygma-protocol-62f126a54b7d5a55203732f0/issues/gh/sygmaprotocol/devops-backlog/25)

Closes: #<issue>

## How Has This Been Tested? Testing details.
[Tested](https://github.com/sygmaprotocol/sygma-relayer/actions/runs/3904350798)
-  Tested - on Pull_request

## Types of changes
- Removed Duplicates files
- Usage of one Runner to replace five runners
- Replaced hard-coded values with secrets variables

### CODE REVIEW

**Comment by MakMuftic** [comment]
Hey @eedygreen , we are using semantic naming for PRs - meaning that each PR should be prefixed with: `fix:`, `feat:` or `chore:`. We use this to generate meaningful changelog from PRs. I would say for this PR appropriate prefix (type) is `chore`.

You can check closed PRs for some more context.

**Comment by rbadillap** [comment]
Hi, there's an [active PR](https://github.com/sygmaprotocol/sygma-relayer/pull/59) here that covers most of the issues mentioned here.

For @MakMuftic: [We also created a ticket (assigned to @eedygreen)](https://github.com/sygmaprotocol/devops-backlog/issues/29) to work on this implementation in other services.

@eedygreen let me merge mine and then you can proceed with yours (you may have some conflicts that need to resolve first)

**Comment by P1sar** [comment]
@eedygreen Please send the Linter Error message to fix the SemanticPR problem 
`Error: Pull request has only one commit and it's not semantic; this may lead to a non-semantic commit in the base branch (see https://github.community/t/how-to-change-the-default-squash-merge-commit-message/1155). Amend the commit message to match the pull request title, or add another commit.`

**Comment by rbadillap** [comment]
[This PR](https://github.com/sygmaprotocol/sygma-relayer/pull/59) is ready to be merged, please make sure to update your branch and make your changes.

Thanks @eedygreen

**Review by P1sar — APPROVED** [review]

File: .github/workflows/deploy_ecs_relayers.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+name: Deploy ECS relayers
```
Comment (rbadillap) [inline .github/workflows/deploy_ecs_relayers.yml:1]: Please consider using the file [deploy_testnet.yml](https://github.com/sygmaprotocol/sygma-relayer/blob/main/.github/workflows/deploy_testnet.yml), which is the modified file I've merged in the mentioned PR above.

In that file you'll find some other changes I've implemented, such as: `ECR_REPOSITORY` is now:  `relayer`, the docker tag to be pushed (TAG) is now: `stable`, and also the [use of matrix](https://github.com/sygmaprotocol/sygma-relayer/blob/main/.github/workflows/deploy_testnet.yml#L54) to deploy all the relayers in the same workflow.

**Review by rbadillap — CHANGES_REQUESTED** [review]
In addition to taking a look at the file [deploy_testnet.yml](https://github.com/sygmaprotocol/sygma-relayer/blob/main/.github/workflows/deploy_testnet.yml) (which is the file I've modified in the previous merged PR), we need to do the same changes in [deploy_stage.yml](https://github.com/sygmaprotocol/sygma-relayer/blob/main/.github/workflows/deploy_stage.yml) which is the other environment.

The difference between testnet and stage in GitHub Actions is that we trigger Stage when a code is pushed into main, the docker tag is `latest` and the ECR_REPOSITORY needs be moved to `relayers` as testnet to have everything in a single ECR repo.

**Review by MakMuftic — APPROVED** [review]
LGTM

**Review by mpetrun5 — APPROVED** [review]

**Review by rbadillap — APPROVED** [review]
Good catch to move ECS task definitions to another repo!

**Pull request merged**